### PR TITLE
Update to TensorFlow v1.13

### PR DESCRIPTION
**Pull request by reuben**

### CODE REVIEW

**Comment by reuben** [comment]
Mostly green, just needs the Dockerfile to be updated and the Windows CUDA worker update to be deployed.

**Comment by lissyx** [comment]
@reuben CUDA v10.0 has been now properly redeployed, I've retriggered in https://tools.taskcluster.net/groups/B2QqIbPgQ7SBXl7GB4HTCA/tasks/K-4g1DZzTTO6O637PBOYpw/runs/0

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 pandas
 progressbar2
 python-utils
-tensorflow == 1.12.0
+tensorflow == 1.13.1
```
Comment (lissyx) [inline requirements.txt]: Pretty sure there are other places this is required

**Comment by lissyx** [comment]
@9define Upstream seems to recomment 10.0 for now

File: taskcluster/.shared.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 python:
   packages_trusty:
-    apt: 'make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev'
+    apt: 'make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev libhdf5-dev'
```
Comment (lissyx) [inline taskcluster/.shared.yml]: So I guess you did that because of the failed Python tests. However, I'm not sure we want to have to build `tables` from source, it might delay quite a lot the process. And the lack of 3.4 wheel might be an upstream issue.

I've opened https://github.com/mozilla/DeepSpeech/pull/1961 and https://github.com/PyTables/PyTables/issues/727, it might be better we don't pull `libhdf5-dev` then.

**Comment by lissyx** [comment]
@9define Straight from the NVIDIA forums: https://devtalk.nvidia.com/default/topic/1047898/cuda-10-1-tensorflow-1-13
tl;dr "TensorFlow r1.13 breaks with CUDA 10.1"

File: taskcluster/.shared.yml (hunk 2 above)
Comment (lissyx) [reply taskcluster/.shared.yml]: @reuben So I'd suggest we merge #1961 and remove this commit :)
